crates/ruma-client-api/src/error.rs

```diff
@@ -369,29 +388,32 @@ impl std::error::Error for Error {}
 impl ErrorBody {
     /// Convert the ErrorBody into an Error by adding the http status code.
```

Comment:
Should we add a note that this is equivalent to the other constructor? And potentially in the other direction too?

Reply:
Also having a note on `Error::new` that points to `ErrorBody::into_error` as an alternative spelling of the same thing.